support-frontend/setup.sh

```diff
@@ -115,6 +117,14 @@ install_awscli() {
   fi
 }
 
+install_dev_nginx() {
+  if ! installed dev-nginx; then
+    brew tap "guardian/devtools"
```

Comment:
I think a `Brewfile` could make the setup simpler. A `Brewfile` is essentially a package.json for brew.

For example, we added it to one of our Tools recently - https://github.com/guardian/login.gutools/pull/44/files.

The simpler setup then reads something like `brew bundle` (to install deps) `yarn install && yarn build` `sbt run`.

Reply:
This change is probably too much for this PR though.